MANIFEST.in

```diff
@@ -3,4 +3,5 @@ recursive-include src/snowflake/connector *.py *.pyx
 include LICENSE.txt
 include NOTICE
 recursive-include src/snowflake/connector/cpp *.cpp *.hpp
+include src/snowflake/connector/antl *.g *.py
```

Comment:
I think this space here doesn't work as you intended it to.
I think you meant to copy the line above [recursive-include works differently than include](https://packaging.python.org/guides/using-manifest-in/#manifest-in-commands)